dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -107,6 +113,15 @@ object GlobalRum {
 
     // region Internal
 
+    internal fun addUserInteraction() {
+        val nanoTime = System.nanoTime()
+        val isInactiveSession = (nanoTime - lastUserInteractionNs.get()) >= SESSION_INACTIVITY_NS
+
+        if (!isInactiveSession) {
+            lastUserInteractionNs.set(System.nanoTime())
```

Comment:
if we want to update the lastUserInteraction....shouldn't we be doing this no matter what ?  I don't think you need the `if` here.

Reply:
If the session is inactive, we need to renew the sessionId. Updating the last user action would extends the expired session duration

Reply:
oh...my bad it was a (!) in there ...sorry...it makes sense then.